foundational-cxx-support-matrix.md

```diff
@@ -1,26 +1,48 @@
 # Foundational C++ Support
 
-| Dimension       | Supported Version      | Last Changed | Next Change [^next-change] |
+The relevant policies are described at https://opensource.google/documentation/policies/cplusplus-support.
```

Comment:
If we move this up for C++, can you do this for the other tables as well to keep this consistent?